src/Razor/src/Microsoft.AspNetCore.Razor.LanguageServer/CodeActions/ILSPCodeActionResolverHandler.cs

```diff
@@ -0,0 +1,15 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using Microsoft.AspNetCore.Razor.LanguageServer.CodeActions.Models;
+using OmniSharp.Extensions.JsonRpc;
+
+namespace Microsoft.AspNetCore.Razor.LanguageServer.CodeActions
+{
+    [Serial, Method("textDocument/codeActionResolve")]
+    internal interface ILSPCodeActionResolverHandler :
```

Comment:
Given this is a VS specific spec maybe we should indicate that in the name:
```suggestion
    internal interface IVSCodeActionResolverHandler :
```

Reply:
I was trying to avoid that simply because it could make the naming a bit confusing. 

It is a `VSCode ActionResolverHandler` or `VS CodeActionResolverHandler`.

How about:
```suggestion
    internal interface IVSLSPCodeActionResolverHandler :
```

Reply:
It's VS isn't it? Because this textDocument/codeActionResolve endpoint is only accessible via VS?